### ISSUE TITLE
[Tensorboard] nothing appears in the projector tab

### ISSUE THREAD

**Issue by jc-hou** [pytorch/pytorch#30525]
## 🐛 Bug

When running the add_embedding example provided in the official [documentation](https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_embedding), I am unable to visualize the output in Tensorboard. When loading the projector page, it's blank. Yet the image and graph tab both works fine.

![tb_200](https://user-images.githubusercontent.com/30210529/69751901-51768880-1150-11ea-8db8-0ba4be404f8d.png)

## Script To Reproduce

`import keyword
import torch

import torch
import torchvision
from torch.utils.tensorboard import SummaryWriter
from torchvision import datasets, transforms

# Writer will output to ./runs/ directory by default
writer = SummaryWriter()

transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.5,), (0.5,))])
trainset = datasets.MNIST('mnist_train', train=True, download=True, transform=transform)
trainloader = torch.utils.data.DataLoader(trainset, batch_size=64, shuffle=True)
model = torchvision.models.resnet50(False)
# Have ResNet model take in grayscale rather than RGB
model.conv1 = torch.nn.Conv2d(1, 64, kernel_size=7, stride=2, padding=3, bias=False)
images, labels = next(iter(trainloader))

grid = torchvision.utils.make_grid(images)
writer.add_image('images', grid, 0)
writer.add_graph(model, images)

meta = []
while len(meta)<100:
    meta = meta+keyword.kwlist # get some strings
meta = meta[:100]

for i, v in enumerate(meta):
    meta[i] = v+str(i)

label_img = torch.rand(100, 3, 10, 32)
for i in range(100):
    label_img[i]*=i/100.0

writer.add_embedding(torch.randn(100, 5), metadata=meta, label_img=label_img)
writer.add_embedding(torch.randn(100, 5), label_img=label_img)
writer.add_embedding(torch.randn(100, 5), metadata=meta)

writer.close()`

## Environment
pytorch: 1.3.0
torchvision: 0.5.0a0+9cdc814
tensorboard: 2.0.0
tensorflow not installed.

**Comment by lanpa** [comment]
Hi @jc-hou , I tried your code with torch==1.3.0 and use tensorboard==2.1.0. The page load correctly.
My logdir: [runs.zip](https://github.com/pytorch/pytorch/files/3945977/runs.zip)

**Comment by bsun0802** [comment]
I have a similar bug here with torch==1.3.1 and tensorboard==2.1.0. I copy-and-paste the code from the [official tutorial](https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html), but the projector tab does not appear, the graph and image works well, but not for projectors.

**Comment by lanpa** [comment]
@bsun0802 Have you tried to select projectors from the drop-down list on the top right?

**Comment by silence401** [comment]
Aight, thanks.

**Comment by ProKil** [comment]
I can reproduce this issue on Safari, but not on Chrome. On Safari, the error message says "path /data/index.js not found, sending 404". If you encounter the same problem, you can try different browsers.

**Comment by lanpa** [comment]
@ProKil Thanks, I can reproduce it too with safari.

**Comment by Joxis** [comment]
I had the same issue with torch==1.4.0 and tensorboard==2.0.0 on Windows. The projector tab was blank in Chrome, Firefox and Edge. In Chrome the following message popped up in the console:
```
Failed to load module script: The server responded with a non-JavaScript MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.
```

In my case the issue was caused by a bad MIME type as described in [this TensorBoard issue](https://github.com/tensorflow/tensorboard/issues/3117). The proposed register solution fixed it for me.

**Comment by rbudnar** [comment]
I had this issue as well on windows. Updating to tb-nightly (2.2.0a20200214) resolved the blank projector issue for me.

**Comment by neo4dev** [comment]
I had this issue as well, and the solution is to restart tensorboard in the terminal

**Comment by arisliang** [comment]
same issue on windows 10

torch: 1.5.0
tensorboard: 2.2.1

**Comment by nihuo76** [comment]
same issue on windows 10

torch: 1.4.0
tensorboard: 2.2.1

**Comment by LucaMarconato** [comment]
same issue on CentOS
```
pytorch                   1.6.0           py3.7_cuda9.2.148_cudnn7.6.3_0
tensorboard               2.3.0
tensorboard-plugin-wit    1.7.0
torch-cluster             1.5.7                    
torch-geometric           1.6.1                 
torch-scatter             2.0.5                   
torch-sparse              0.6.7                   
torch-spline-conv         1.2.0                
torchfile                 0.1.0                      
torchvision               0.7.0                 py37_cu92
```

**Comment by ixsluo** [comment]
Using Chrome instead of Safari also fixed this issue for me.

- torch: 1.11.0
- tensorboard: 2.8.0
- tensorboardX: 2.5

The error message is "path /data/index.js not found, sending 404" when using Safari.
